cmd/agent/gui/checks_pup.go

```diff
@@ -0,0 +1,11 @@
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build !cpython
+
+package gui
+
+// stub: no python interprter
+func getPythonChecks() ([]string, error) {
+	return []string{}, nil
```

Comment:
should we return nil instead of an empty string slice ?

Reply:
I actually think this might be safer to use as error is `nil`, user would typically check the return error. I could also return `nil` and set the error to something related to the puppy agent.